content/applications/hr/employees/new_employee.rst

```diff
@@ -12,19 +12,20 @@ appears. Fill out the required information and any additional details.
 
 .. note::
    The current company phone number and name is populated in the :guilabel:`Work Phone` and
-   :guilabel:`Company` fields.
+   :guilabel:`Company` fields. If the *Appraisals* application is installed, the :guilabel:`Next
+   Appraisal Date` field is populated with a date six months from today's date.
 
 General information
 ===================
 
-The employee form automatically saves as data is entered, but the save manually icon (a could with
+The employee form automatically saves as data is entered, but the save manually icon (a cloud with
```

Comment:
I only disagree with this since the cloud does not have the arrow in it, and I think that is a very important part of the description, and the icon might confuse the reader. Since I need to add the text abut the arrow, I will keep it all text.